src/layout/dom.ts

```diff
@@ -1,9 +1,14 @@
+import { Priority } from "@clarity-types/core";
 import { Constant, NodeChange, NodeInfo, NodeValue, Source } from "@clarity-types/layout";
 import config from "@src/core/config";
+import { schedule } from "@src/core/task";
 import time from "@src/core/time";
+import discover from "@src/layout/discover";
 import selector from "@src/layout/selector";
 
 let index: number = 1;
+const BLACKLIST_TYPES = ["password", "hidden", "email"];
+const BLACKLIST_NAMES = ["address", "cell", "code", "dob", "email", "mobile", "name", "phone", "secret", "social", "ssn", "tel", "zip"];
```

Comment:
This is just a heuristic, correct? None of these has any "official" meaning.

Reply:
Yes right.

Reply:
Please add comment about that.

---
In reply to: [406557724](https://github.com/microsoft/clarity-js/pull/172#discussion_r406557724) [](ancestors = 406557724)